testing/architecture-test/src/changes/archunit-store/internal-api-nullability.txt

```diff
@@ -1410,6 +1410,41 @@ Class <org.gradle.external.javadoc.internal.OptionLessStringsJavadocOptionFileOp
+Class <org.gradle.fileevents.internal.FileEventsVersion> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (FileEventsVersion.java:0)
+Class <org.gradle.fileevents.internal.InotifyInstanceLimitTooLowException> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (InotifyInstanceLimitTooLowException.java:0)
+Class <org.gradle.fileevents.internal.InotifyWatchesLimitTooLowException> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (InotifyWatchesLimitTooLowException.java:0)
+Class <org.gradle.fileevents.internal.InsufficientResourcesForWatchingException> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (InsufficientResourcesForWatchingException.java:0)
+Class <org.gradle.fileevents.internal.LinuxFileEventFunctions$LinuxFileWatcher> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (LinuxFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.LinuxFileEventFunctions$WatcherBuilder> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (LinuxFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.LinuxFileEventFunctions> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (LinuxFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.NativeLogger$LogLevel> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (NativeLogger.java:0)
+Class <org.gradle.fileevents.internal.NativeLogger> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (NativeLogger.java:0)
+Class <org.gradle.fileevents.internal.OsxFileEventFunctions$OsxFileWatcher> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (OsxFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.OsxFileEventFunctions$WatcherBuilder> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (OsxFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.OsxFileEventFunctions> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (OsxFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.WindowsFileEventFunctions$WatcherBuilder> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (WindowsFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.WindowsFileEventFunctions$WindowsFileWatcher> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (WindowsFileEventFunctions.java:0)
+Class <org.gradle.fileevents.internal.WindowsFileEventFunctions> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (WindowsFileEventFunctions.java:0)
```

Comment:
💭 The plan is to use JSpecify's `@NulLMarked` in `gradle-fileevents` (see gradle/gradle-fileevents#14), but let's wait until Gradle itself adopts it. Once we adopt JSpecify, we can remove these exceptions.